website/content/2021/min-degree.md

```diff
@@ -0,0 +1,222 @@
++++
+# The title of your blogpost. No sub-titles are allowed, nor are line-breaks.
+title = "A story about min-degree ordering"
+# Date must be written in YYYY-MM-DD format. This should be updated right before the final PR is made.
+date = 2021-09-27
+
+[taxonomies]
+# Keep any areas that apply, removing ones that don't. Do not add new areas!
+areas = ["Algorithm", "Graph Theory"]
+# Tags can be set to a collection of a few keywords specific to your blogpost.
+# Consider these similar to keywords specified for a research paper.
+tags = ["Min-degree ordering", "Gaussian Elimination"]
+
+[extra]
+# For the author field, you can decide to not have a url.
+# If so, simply replace the set of author fields with the name string.
+# For example:
+#   author = "Harry Bovik"
+# However, adding a URL is strongly preferred
+author = {name = "Junxing Wang", url = "junxingwang.org" }
```

Comment:
Also, I just noticed that clicking on your name does not go to your website as expected, since you have not specified your full URL. Please fix it:

```suggestion
author = {name = "Junxing Wang", url = "https://junxingwang.org/" }
```